python/ciqueue/distributed.py

```diff
@@ -70,14 +72,14 @@ def poll():
             while not self.shutdown_required and len(self):  # pylint: disable=len-as-condition
                 test = self._reserve()
                 if test:
-                    yield test
+                    yield test.decode()
                 else:
                     time.sleep(0.05)
 
         try:
             self.wait_for_master()
             for i in poll():
-                yield i
+                yield i if isinstance(i, str) else i.decode()
```

Comment:
I suspect that I added this and it isn't required. From inspection, it seems `i` will always be decoded on line 75.

Reply:
:+1: K, I'll try it out.